llvm/include/llvm/Support/ScopedPrinter.h

```diff
@@ -220,59 +220,59 @@ class LLVM_ABI ScopedPrinter {
   }
 
   virtual void printNumber(StringRef Label, char Value) {
-    startLine() << Label << ": " << static_cast<int>(Value) << "\n";
+    startLine() << Label << ": " << static_cast<int>(Value) << '\n';
```

Comment:
Hm, I believe the `<< "\n"` pattern is considered more idiomatic. grep says it's much more prevalent in the LLVM codebase (by a factor of 2.4).

Reply:
I stared https://discourse.llvm.org/t/micro-coding-standard-question/88469 as well, just to get a consensus here.